conf/application.conf

```diff
@@ -140,6 +140,10 @@ auditing {
 
 feature-switch {
   allowTemporalValidationSuspension.enabled = true
+  cl402 {
```

Comment:
Minor point up for discussion. Not sure what the consensus is on this but eventually these switches will be turned on everywhere and ideally we want to leave the environment configs tidy (without old switches hanging around) when that happens. So I think it's probably safest to keep the switches as true locally, so that they can be removed from the environment configs when they are enabled. This will also reduce the likelihood of accidentally turning off something when the environment configs are tidied up based on this assumption. However setting released-in-production to true here might confuse so maybe don't even mention the cl402  switch in this file at all (and leave is one fewer thing to tidy up). WDYT?